provider/server/machine_volume_attach_test.go

```diff
@@ -36,7 +37,7 @@ var _ = Describe("AttachVolume", func() {
 
 		DeferCleanup(func(ctx SpecContext) {
 			Eventually(func() bool {
-				_, err := machineClient.DeleteMachine(ctx, &iri.DeleteMachineRequest{MachineId: createResp.Machine.Metadata.Id})
+				_, err := machineClient.DeleteMachine(ctx, &machineiriv1alpha1.DeleteMachineRequest{MachineId: createResp.Machine.Metadata.Id})
 				Expect(err).To(SatisfyAny(
 					BeNil(),
 					MatchError(ContainSubstring("NotFound")),
```

Comment:
i recommend extend this test with check of filesystem. We will need to check, if folder for machine is deleted when machine was deleted. We have this logic in different test.